react/week2/to-do-app/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import './App.css';
+import ToDoList from './ToDoList';
+
+const todoList = [
+  {id: 1,description: "Get out of bed"},
+  {id: 2,description: "Brush teeth"},
+  {id: 3,description: "Eat breakfast"},
+];
+let counter=todoList.length+1;
+
+function App() {
+  const onButtonClick = () => {
+    let newToDo={};
+    newToDo.id=counter++;
+    newToDo.description='Random ToDo';
```

Comment:
I would prefer doing this in just 1 step:
```suggestion
    const newToDo={
      id: counter++,
      description: 'Random ToDo'
    };
```

Reply:
Thank you, i have resolved this part of code